### DISCUSSION TITLE
Add Custom Role support for granular control of user permissions

### DISCUSSION THREAD

**Discussion by DarkWiggerl** [dani-garcia/vaultwarden#3480] [Ideas]
Please add the Feature of "Custom Roles" for granular control of user permissions. 

This is the only way to "Adminsitrate Organisations without access to the Passwords". 

Details about the Function here https://bitwarden.com/help/article/user-types-access-control/#custom-role 

Thanks!! :)

**Comment by BlackDex** [comment]
We do accept well written PR's 😉

**Reply by ghost** [reply]
If only i knew how to write code I would :(
